.github/workflows/pull_requests.yml

```diff
@@ -33,8 +37,22 @@ jobs:
       if: success()
       run: npx grunt prod
 
+    - name: Production Image Build
+      if: success()
```

Comment:
I realise this exists elsewhere, but `if: success()` is the default value, so we shouldn't need to add this.

Reply:
Yea thats what I thought too, didn't want to against the norm! Will switch it up when I do the multi stage build.